package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "if",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare": "husky install",
+    "prettier": "prettier --write src/**/*.js",
```

Comment:
Prettier will be work for src directory, however there are not src. And **you dont't have main.js and index.html.**

Please **add** main.js and index.html and **check** working of prettier and eslint